fileresult.txt

```diff
@@ -0,0 +1 @@
+writer: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=a9fa6ee4b9e6c4a925376df3495c3004a5c07c9a, for GNU/Linux 3.2.0, with debug_info, not stripped
```

Comment:
@Parthvarsani13 it looks like this wasn't created with your cross compiler, it's an x86-64 executable.